src/components/tour/TourHelper.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { connect } from 'redux-bundler-react'
+
+export const TourHelper = ({ doEnableTours, size = 23 }) => {
+  const handleClick = () => {
+    doEnableTours()
+  }
+
+  return (
+    <span className='dib mr1 ml4-m pointer' onClick={handleClick}>
```

Comment:
any reason not to pass `doEnabledTours` in directly to the `onClick`

Reply:
Nope. I personally always prefer to have a separate function so it's easier to add more logic in the future.